packages/mgt-components/src/components/mgt-search-results/mgt-search-results.ts

```diff
@@ -0,0 +1,1239 @@
+        </div>
+        <div class="search-result-content">
+          <div class="search-result-name">
+            ${
+              resourceUrl
+                ? html`
+                  <a href="${resourceUrl}?Web=1" target="_blank">${this.getResourceName(resource)}</a>
+                `
+                : html`
+                  ${this.getResourceName(resource)}
+                `
+            }            
+          </div>
+          <div class="search-result-summary" .innerHTML="${this.getResultSummary(result)}"></div>
+        </div>  
+      </div>          
+      <fluent-divider></fluent-divider>
+    `;
+  }
+
+  /**
+   * Gets default resource URLs
+   * @param resource
+   */
+  private getResourceUrl(resource: SearchResource) {
+    return resource.webUrl || /*resource.url ||*/ resource.webLink || null;
+  }
+
+  /**
+   * Gets default resource Names
+   * @param resource
+   */
+  private getResourceName(resource: SearchResource) {
+    return resource.displayName || resource.subject || trimFileExtension(resource.name);
+  }
+
+  /**
+   * Gets default result summary
+   * @param resource
+   */
+  private getResultSummary(result: SearchHit) {
+    return sanitizeSummary(result.summary || (result.resource as SearchResource)?.description || null);
+  }
+
+  /**
+   * Gets default resource icon
+   * @param resource
+   */
+  private getResourceIcon(resource: SearchResource) {
+    switch (resource['@odata.type']) {
+      case '#microsoft.graph.site':
+        return getSvg(SvgIcon.Globe);
+      case '#microsoft.graph.message':
+        return getSvg(SvgIcon.Email);
+      case '#microsoft.graph.event':
+        return getSvg(SvgIcon.Event);
+      case 'microsoft.graph.chatMessage':
+        return getSvg(SvgIcon.SmallChat);
+      default:
+        return getSvg(SvgIcon.FileOuter);
+    }
+  }
+
+  /**
+   * Validates if cache should be retrieved
+   * @returns
+   */
+  private shouldRetrieveCache(): boolean {
+    return getIsResponseCacheEnabled() && this.cacheEnabled && !this.isRefreshing;
+  }
+
+  /**
+   * Validates if cache should be updated
+   * @returns
+   */
+  private shouldUpdateCache(): boolean {
+    return getIsResponseCacheEnabled() && this.cacheEnabled;
+  }
+
+  /**
+   * Builds the appropriate RequestOption for the search query
+   * @returns
+   */
+  private getRequestOptions(): SearchRequest | BetaSearchRequest {
+    var requestOptions: SearchRequest = {
+      entityTypes: this.entityTypes as EntityType[],
+      query: {
+        queryString: this.queryString
+      },
+      from: this.from ? this.from : undefined,
+      size: this.size ? this.size : undefined,
+      fields: this.getFields(),
+      enableTopResults: this.enableTopResults ? this.enableTopResults : undefined
+    };
+
+    if (this.entityTypes.includes('externalItem')) {
+      requestOptions.contentSources = this.contentSources;
+      /*requestOptions.resultTemplateOptions = {
+        enableResultTemplate: true
+      };*/
```

Comment:
For now, yes. I'll remove and re-add when we attack the Adaptive Cards rendering.